cmd/nucleus/bin.go

```diff
@@ -139,8 +138,6 @@ func run(cmd *cobra.Command, args []string) {
 		logger.Fatalf("failed to initialize cache manager: %v", err)
 	}
 
-	parserService := parser.New(tcm, logger)
```

Comment:
why we removing parsing pod ? Please add jira ticket and fill necessary details.

Reply:
this is still a WIP. Therefore, had marked it as Draft